frontend/src/components/organisms/SignupForm.tsx

```diff
@@ -96,10 +101,15 @@ const SignupForm = () => {
       // Log in
       const { email, password } = data;
       const signedInUser = await signInWithEmailAndPassword(email, password);
+      const emailSent = await sendEmailVerification(); // Send email verification
```

Comment:
Hmmm so I think I only received the email after I clicked on the resend email verification in the verify page. I didn't automatically get it after signin up. Are u able to reproduce?

Reply:
We weren't able to reproduce this error; when we go to the verify page we automatically got the email.